### PR TITLE
CR-1095840 ASTeR TC 7.01 - u50 nodma - Validate errors with "No DMA b…

### PR DESCRIPTION
**Pull request by sadasivanshaiju**
…uffers could be allocated."

### CODE REVIEW

File: src/runtime_src/core/tools/xbutil2/SubCmdValidate.cpp [hunk 1]
```diff
@@ -820,6 +820,17 @@ dmaTest(const std::shared_ptr<xrt_core::device>& _dev, boost::property_tree::ptr
...
 
+  std::vector<std::string> dma_thr ;
+
+  try {
+   dma_thr = xrt_core::device_query<xrt_core::query::dma_threads_raw>(_dev);
+  } catch(...){}
+
+  if (dma_thr.size() == 0){
+    _ptTest.put("status", "skipped");
+    return ;
```
Comment (AShivangi) [inline src/runtime_src/core/tools/xbutil2/SubCmdValidate.cpp:831]: Indentation: there are 3 spaces, it should be only 2

**Review by AShivangi — APPROVED** [review]
A minor comment to fix indentation. Please merge after this is fixed

**Comment by dayeh-xilinx** [comment]
retest this please. Build failed on a network glitch. Retry.

**Comment by dayeh-xilinx** [comment]
retest this please. Timed out again when downloading docker image from artifactory during build stage. Try again.

**Pull request merged**